### ISSUE TITLE
Test and complete User Journeys with VoiceOver for Mac

### ISSUE THREAD

**Issue by developpeurweb** [SAP/spartacus#7829]
Improving **VoiceOver for Mac** navigation skills by completing User Journeys. This is part of Master Ticket GH-7423.

**Guides and details:**
https://cxwiki.sap.com/display/spar/Screen+Readers

**General:**

- [x] Crash recon.
- [x] Try all commands from the guide.
- [x] Checkout product by hearing and watching.
- [x] Checkout product by hearing only.
- [x] How to demo: Test Zoom, alt SnagIt.
- [x] Spot browser differences: Safari/Chrome/FF.
- [x] How to create tickets? e.g. Attach current video/sound, propose new narration.

**How:**

- [x] Lock/Unlock Focus.
- [x] Browse back if clicked by mistake.
- [x] Paragraph/Sentence/Word command difference.

**Contrast:**

- [x] Facets/Filtering.
- [x] Navigate by Next Element Type.
- [x] Rotor content.
- [x] Cart structure.
- [x] Forms structure.
- [x] Tables structure.
- [x] Tables vs List.
- [x] Validation patterns.
- [x] Narration.
- [x] Shopping experience.

**Checkout:**

- [x] By exploration (full narration).
- [x] By Rotor.
- [x] By Next Element Type.
- [x] Expected narration transcript (draft).

**Comment by developpeurweb** [comment]
- Unable to checkout products by hearing.
- No browser differences found.

Nothing to QA for this ticket.
